src/main/java/com/myhome/domain/CommunityAmenity.java

```diff
@@ -0,0 +1,40 @@
+package com.myhome.domain;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import java.time.LocalDate;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class CommunityAmenity extends BaseEntity {
+
+  @Column(nullable = false, unique = true)
+  private String amenityId;
+  @Column(nullable = false)
+  private String discription;
+  @Column
+  private boolean isBooked;
+  @Column
+  private LocalDate bookingStartDate;
```

Comment:
This ideally has to be Date and Time.
Use case: HouseMember books a tennis court from 10 AM to 11 AM on 01-01-2020.

Reply:
frontend can convert `01.01.2020 10 AM` to `01.01.2020 22:00:00`?